electrumutils.py

```diff
@@ -0,0 +1,212 @@
+import logging
+import binascii
+from utils import BITCOIND_CONFIG
+import time
+import sys
+import os
+import tempfile
+import traceback
+import asyncio
+from electrumx.server.controller import Controller
+from electrumx.server.env import Env
+import threading
+from electrum.daemon import Daemon, get_fd_or_server
+from electrum import simple_config, constants, keystore, util
+from electrum.wallet import Wallet
+from electrum.storage import WalletStorage
+from electrum.mnemonic import Mnemonic
+
+bh2u = lambda x: binascii.hexlify(x).decode('ascii')
+
+class ElectrumX:
+    def __init__(self):
+        print("server/__init__")
```

Comment:
Please use the logging framework for messages, otherwise we can't control the output and it'll be hard to tune to find issues.